controllers/toplevel/index.js

```diff
@@ -6,13 +6,17 @@ const rp = require('request-promise');
 const ab = require('express-ab');
 const { sortBy } = require('lodash');
 const xss = require('xss');
+const Raven = require('raven');
+const _ = require('lodash');
```

Comment:
I don't see lodash being used anywhere in this file?

Reply:
Ah yeah, was using it and forgot to remove it. Normally my IDE alerts me to unused variables, but in this case, the variable name `_` made the error underline quite difficult to spot...
![image](https://user-images.githubusercontent.com/394376/31812183-16c9681c-b57a-11e7-97f2-6d267ba40def.png)